test/test_wacom.py

```diff
@@ -241,3 +242,67 @@ def axval(axis: str) -> int:
                 assert first[name] < current[name], f"for axis {name}"
             else:
                 assert first[name] == current[name], f"for axis {name}"
+
+
+def test_scroll(mainloop, opts):
+	"""
+	Check panscrolling works correctly
+	"""
+	dev = Device.from_name("PTH660", "Pen")
+	opts["Button"] = "2 pan"
+	opts["PanScrollThreshold"] = "150"
+
+
```

Comment:
`black` should complain about this extra line here :) and a few other style issues in this file